src/app/Instance/InstancePage/ProcessorsTabContent.tsx

```diff
@@ -142,6 +137,20 @@ export const ProcessorsTabContent = ({
     error: processorsError,
   } = useGetProcessorsApi();
 
+  const customToolbarElement = areProcessorsLoading ? (
+    <Skeleton width={"170px"} height={"35px"} />
+  ) : bridgeStatus == ManagedResourceStatus.Failed ? (
+    <Button ouiaId="create-processor" variant="primary" isDisabled={true}>
+      {t("processor.createProcessor")}
+    </Button>
+  ) : (
+    <Link to={`/instance/${instanceId}/create-processor`}>
+      <Button ouiaId="create-processor" variant="primary">
+        {t("processor.createProcessor")}
+      </Button>
+    </Link>
+  );
```

Comment:
Generally speaking, I'm not too fond of using two chained ternary operators. Moreover, we can avoid repeating the whole `Button` definition with just one property (`isDisabled`) that could depend on a variable.